Hydra/ApiDocumentationBuilder.php

```diff
@@ -134,10 +142,10 @@ public function getApiDocumentation()
                     'hydra:property' => [
                         '@id' => ($iri = $attributeMetadata->getIri()) ? $iri : sprintf('#%s/%s', $shortName, $attributeName),
                         '@type' => $type,
-                        'rdfs:label' => $attributeName,
+                        'rdfs:label' => $convertedName,
```

Comment:
I only reflect what has been done in http://www.markus-lanthaler.com/hydra/api-demo/vocab (search for `vocab:User/raisedIssues`).
The `hydra:title` is used in HydraConsole to display the property in the documentation panel on the right, or the form labels when trying to make a POST operation for instance.

Also, there is a problem using NelmioApiDocBundle: the properties aren't converted with the converter neither.

What do you think ?